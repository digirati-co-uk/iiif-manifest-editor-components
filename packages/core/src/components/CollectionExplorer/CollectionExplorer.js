import React from 'react';
import PropTypes from 'prop-types';

const CollectionExplorer = ({ url }) => (
  <div>Collection explorer placeholder</div>
);

CollectionExplorer.propTypes = {
  url: PropTypes.string,
};

CollectionExplorer.defaultProps = {
  url: '',
};

export default CollectionExplorer;
