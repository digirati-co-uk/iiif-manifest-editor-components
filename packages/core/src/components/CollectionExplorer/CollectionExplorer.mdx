---
name: Collection Explorer
menu: Components
---

import { Playground, PropsTable } from 'docz';
import CollectionExplorer from './CollectionExplorer'; 

# Collection Explorer

<PropsTable of={CollectionExplorer} />