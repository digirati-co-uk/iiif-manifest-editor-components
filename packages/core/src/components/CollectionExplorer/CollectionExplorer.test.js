import React from 'react';
import CollectionExplorer from './CollectionExplorer';

describe('CollectionExplorer', () => {
  it('renders the collection explorer unchanged', () => {
    const collectionExplorer = shallow(<CollectionExplorer url={null} />);
    expect(collectionExplorer).toMatchSnapshot();
  });
});
