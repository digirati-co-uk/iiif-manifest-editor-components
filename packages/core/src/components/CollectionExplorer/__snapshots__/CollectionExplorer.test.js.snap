// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionExplorer renders the collection explorer unchanged 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CollectionExplorer
    url={null}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Collection explorer placeholder",
    },
    "ref": null,
    "rendered": "Collection explorer placeholder",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Collection explorer placeholder",
      },
      "ref": null,
      "rendered": "Collection explorer placeholder",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
